docs/installation/gateways.md

```diff
@@ -0,0 +1,345 @@
+
+`https://bzz.link`
+
+## Limitations
+
+### Rate and Filesize Limits
+
+To ensure everyone is able to try out the Swarm network, rate and filesize limitations are imposed on these public gateways. If you require that yourself, or the users of your Dapp require access which exceeds these limits, you must *run your own gateway*.
+
+### Postage Stamps
+
+When uploading chunks to Swarm, you must provide each with an attached signed postage stamp. Using our public gateway or API, chunks are automatically stamped with random batches that last for about 2 weeks. There is no way to extend or enlarge these batches, or to pick which one you will use.
+
+If you want to be able to control the amount of time your content stays in Swarm, you must have knowledge of your *Batch ID*. In order to choose which chunks are stamped with which batches, you must have access to the relevant key material. Since browser based chunk stamping is not yet implemented in javascript, Gateway Proxy must handle stamping chunks.
+
+## Run Your Own Gateway
+
+Swarm does not yet have a browser based lightnode component, although this feasible and is planned for the future.
+
+In the meantime, in order to access Swarm directly from all your devices, it is possible for you to set use a HTTP gateway. This is similiar to using an RPC provider like *Infura* or *GetBlock* instead of running a blockchain node.
+
+There are three main ways to run a gateway:
+
+### Personal
+
+A personal gateway is to provide Swarm for you and your trusted colleagues, friends and family. This should be protected so that only authenticated users can access it. You can do this by enabling token based authentication, or in the safety of a private network.
+
+### Public
+
+A public gateway is provided for the general public to access any data from Swarm. Only people and institiutions that are aware of the potential risks of allowing anyone to access a Swarm gateway they operate should attempt this.
+
+### Dapp Provider Gateway
+
+Deploy your Dapp to a Web2 domain and provide an associated swarm gateway. You can use CORS to block* access from other websites using your gateway.
+
+To deal with data persistence, there are three approaches which can be chosen based on how long you need your user's content to stay available in the network:
+
+- enable automatic stamping for short lived files
+- enable automatic stamping using the same large batch, by setting a default `POSTAGE_STAMP`
+- pass any postage batch id as a header. the postage batch must belong to your attached Bee node.
+
+* this is only blocked at the browser level, you may want to otherwise authenticate your users sessions or limit access to a subset of the swarm network.
+** we are currently investigating auto top up for the default stamp, for now please manage this manually using *swarm-cli* or *swarm desktop*.
```

Comment:
auto top-up? 
per-user stamp ids?

Reply:
I would remove these two sentences. We can create github issues instead and see if there is an interest in implementing them.